pkg/services/object/acl/basic_helper.go

```diff
@@ -121,6 +121,21 @@ func (a basicACLHelper) SystemAllowed(op eacl.Operation) bool {
 	return false
 }
 
+// InnerRingAllowed returns true if inner ring node as a subset of system group
```

Comment:
InnerRingAllowed returns true if the operation is allowed by ACL for InnerRing nodes, as part of System group.

Reply:
Fixed.